src/backend/distributed/shared_library_init.c

```diff
@@ -450,7 +450,7 @@ RegisterCitusConfigVariables(void)
 		gettext_noop("Sets the maximum duration to connect to worker nodes."),
 		NULL,
 		&NodeConnectionTimeout,
-		5 * MS_PER_SECOND, 10 * MS, MS_PER_HOUR,
+		30 * MS_PER_SECOND, 10 * MS, MS_PER_HOUR,
```

Comment:
We should also change the initializer in `connection_management.c` to avoid confusion.